src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,20 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) => (
+  comments.length !== 0
+    ? (
+      <ul className="CommentList">
+        {comments.map(comment => (
+          <CommentInfo comment={comment} key={comment.id} />
+        ))}
+      </ul>
+    )
+    : (
+      <>
```

Comment:
The hr tag is not semantic HTML. It's better to use CSS for styling. You could replace the hr tag with a border styling in CSS.